support/cas-server-support-oidc/src/main/java/org/apereo/cas/oidc/config/OidcConfiguration.java

```diff
@@ -757,6 +771,29 @@ public View oidcConfirmView() {
         return casProtocolViewFactory.getObject().create(applicationContext, "protocol/oidc/confirm");
     }
 
+    @ConditionalOnMissingBean(name = "oidcSingleLogoutMessageCreator")
+    @Bean
+    public SingleLogoutMessageCreator oidcSingleLogoutMessageCreator() {
+        return new OidcSingleLogoutMessageCreator(buildConfigurationContext());
+    }
+
+    @ConditionalOnMissingBean(name = "oidcSingleLogoutServiceMessageHandler")
+    @Bean
+    public SingleLogoutServiceMessageHandler oidcSingleLogoutServiceMessageHandler() {
+        return new OidcSingleLogoutServiceMessageHandler(httpClient.getObject(),
```

Comment:
You should set an order to this handler runs before the default one if necessary.

Reply:
Sure, I will handle the order aspect of all message handlers.